.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Plugin Name
```

Comment:
Let's change this to be ServiceNow specific, instead of the remnant Plugin field names